docs/tutorial/4-code-update-repeat.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Code. Update. Repeat.
+subtitle: Tilt Tutorial
+layout: docs
+---
+
+We mentioned that Tilt embraces the concept of a [control loop][tutorial-control-loop], so once you've run `tilt up`, it's a "hands free" development experience.
+
+As you edit your code, Tilt will automatically run update steps such as building an updated container image and deploying it.
+
+> 📓 Navigate to the "web" resource in the Tilt UI and "Clear Logs" before continuing
+
+Let's test it out:
+ 1. Open `web/vite.config.js` in your favorite editor
+ 2. Find the `logLevel` line and change it from `'error'` to `'info'`
```

Comment:
i could pooootentially see someone confusing this config change with a setting that has to do with tilt ('cause we've seen the web ui at this point, which has error and info filters). you do explain this later and the focus here is on a file change (not really the content of that file change), so maybe i'm describing a ux edge case.

Reply:
Ahhh that's interesting! I was really struggling to come up with a code change that evades Live Update and landed here, but am not really thrilled with it, as there's not much of an impact.

I wonder if best option might be to intentionally set a specific source code file as a `fall_back_on` for the sake of the tutorial...hmmm